pyfakefs/fake_pathlib.py

```diff
@@ -77,43 +77,43 @@ def _wrapped(pathobj1, pathobj2, *args):
 class _FakeAccessor(pathlib._Accessor):  # pylint: disable=protected-access
     """Accessor which forwards some of the functions to FakeFilesystem methods."""
 
-    stat = _wrap_strfunc(FakeFilesystem.GetStat)
+    stat = _wrap_strfunc(FakeFilesystem.stat)
 
-    lstat = _wrap_strfunc(lambda fs, path: FakeFilesystem.GetStat(fs, path, follow_symlinks=False))
+    lstat = _wrap_strfunc(lambda fs, path: FakeFilesystem.stat(fs, path, follow_symlinks=False))
 
-    listdir = _wrap_strfunc(FakeFilesystem.ListDir)
+    listdir = _wrap_strfunc(FakeFilesystem.listdir)
 
-    chmod = _wrap_strfunc(FakeFilesystem.ChangeMode)
+    chmod = _wrap_strfunc(FakeFilesystem.chmod)
 
     if sys.version_info >= (3, 6):
         scandir = _wrap_strfunc(FakeFilesystem.ScanDir)
 
     if hasattr(os, "lchmod"):
-        lchmod = _wrap_strfunc(lambda fs, path, mode: FakeFilesystem.ChangeMode(
+        lchmod = _wrap_strfunc(lambda fs, path, mode: FakeFilesystem.chmod(
             fs, path, mode, follow_symlinks=False))
     else:
         def lchmod(self, pathobj, mode):
             """Raises not implemented for Windows systems."""
             raise NotImplementedError("lchmod() not available on this system")
 
-    mkdir = _wrap_strfunc(FakeFilesystem.MakeDirectory)
+    mkdir = _wrap_strfunc(FakeFilesystem.makedir)
 
-    unlink = _wrap_strfunc(FakeFilesystem.RemoveFile)
+    unlink = _wrap_strfunc(FakeFilesystem.remove)
 
-    rmdir = _wrap_strfunc(FakeFilesystem.RemoveDirectory)
+    rmdir = _wrap_strfunc(FakeFilesystem.rmdir)
 
-    rename = _wrap_binary_strfunc(FakeFilesystem.RenameObject)
+    rename = _wrap_binary_strfunc(FakeFilesystem.rename)
 
     replace = _wrap_binary_strfunc(lambda fs, old_path, new_path:
-                                   FakeFilesystem.RenameObject(
+                                   FakeFilesystem.rename(
                                        fs, old_path, new_path, force_replace=True))
 
     symlink = _wrap_binary_strfunc_reverse(
         lambda fs, file_path, link_target, target_is_directory:
-        FakeFilesystem.CreateLink(fs, file_path, link_target,
-                                  create_missing_dirs=False))
+        FakeFilesystem.symlink(fs, file_path, link_target,
```

Comment:
This only creates a symbolic link, right?

Reply:
Yes, but I think to rename it to `create_link()` to be consistent with `create_file` and `create_dir`, and also to avoid confusion about the argument order - the arguments here are reversed in comparison to `os.symlink`.

Reply:
Or maybe better `create_symlink`.